code/game/objects/items/robot/robot_upgrades.dm

```diff
@@ -381,3 +381,20 @@
 	R.module.add_module(S, FALSE, TRUE)
 
 	return 1
+	
+/obj/item/borg/upgrade/traitorsec
+        name = "Bloodhound upgrade module"
+        desc = "Ilegal upgrade board that unlocks the banned security module"
+        icon_state = "cyborg_upgrade3"
+        require_module = 1
+        origin_tech = "data=4;combat=4;syndicate=3"
+
+
+/obj/item/borg/upgrade/traitorsec/action(mob/living/silicon/robot/R,/mob/living/user)
+	if(..())
+		return 0
+       
+       wiresexposed = TRUE

```

Comment:
All of these lines should have only one level of indentation, and these wiresexposed lines should be R.wiresexposed